tgui/src/interfaces/ore_redemption_machine.ract

```diff
@@ -46,61 +46,70 @@
 		</ui-section>
 	{{/if}}
 </ui-display>
-<div class="display tabular">
```

Comment:
https://github.com/tgstation/tgstation/pull/39118 this would also be something good to port.